src/test/java/steps/Bing.java

```diff
@@ -0,0 +1,35 @@
+package steps;
```

Comment:
the Page is located in the steps package. To easily identify Page classes I recommend adding *Page in the class name